src/element.py

```diff
@@ -0,0 +1,49 @@
+from periodic_table import PERIODIC_TABLE
+
+
+class Element:
+
+
+    element_dic = {}
+    atomic_num = 0
+    symbol = ''
+    name = ''
+    group = 0
+    period = 0
+    atomic_weight = 0
+    density = 0
+    melting_point = ''
+    boiling_point = ''
+    heat_capacity = ''
+    electronegativity = ''
+    precise = ''
+    type = ''
+
+
+    #constructor
+    def __init__(self,a_num):
+
+        #Stores the dictionary with the info for the specific element
+        self.element_dic = PERIODIC_TABLE[a_num]
+        self.atomic_num = self.element_dic['atomic_num']
```

Comment:
I think its a bit excessive to have them as separate components when we can just have the whole dict saved and access the values as required. We are using double the memory here so we should remove these and just leave the dictionary we get from the table